### ISSUE TITLE
mu42--main-action-str undefined

### ISSUE THREAD

**Issue by BossCode45** [emacs-evil/evil-collection#723]
My init kinda breaks due to this error.
I installed evil collection using use-package, and mu4e from the package "mu" on the AUR (although I edited the package build to remove emacs as a dependency since I compiled and installed manually)
I have evil collection set to run after mu4e.
More information can be provided if you want.
`Error (use-package): evil-collection/:config: Symbol’s function definition is void: mu4e--main-action-str`

**Comment by tshu-w** [comment]
see https://github.com/emacs-evil/evil-collection/issues/695 & https://github.com/emacs-evil/evil-collection/pull/720

**Comment by BossCode45** [comment]
Would you suggest I clone that branch and use that instead?

**Comment by tshu-w** [comment]
> Would you suggest I clone that branch and use that instead?

Yes, if you are using mu > 1.10.x, I have used this PR branch for several days and it works fine.
